examples/adhoc-layout/example/__init__.py

```diff
@@ -1,12 +1,2 @@
-import sys
-
-PY2 = sys.version_info[0] == 2
-
-
-if PY2:
-    def add(a, b):
-        return b + a
-
-else:
-    def add(a, b):
-        return a + b
+def add(a, b):
+    return a + b
```

Comment:
I think this is testing conditional collection and merging - perhaps something like:

```
import platform

if platform.python_implementation() == "PyPy":
    def add(a, b):
        return a + b

else:
    def add(a, b):
        return a + b
```

and then run this example on pypy3.7 and cpython3.9 ?

Reply:
Yep, updated!